source/jormungandr/jormungandr/realtime_schedule/realtime_proxy_manager.py

```diff
@@ -71,7 +72,7 @@ def __init__(self, proxies_configuration):
                 continue
 
             try:
-                rt_proxy = attr(id=proxy_id, **args)  # all services must have an ID
+                rt_proxy = attr(id=proxy_id, code_source=code_source, **args)  # all services must have an ID
```

Comment:
I don't think `code_source` is very explicit. maybe `object_realtime_id_tag` ? or `object_id_tag` ?